features/git-prune-branches/feature_branch/behind_main/without_open_changes.feature

```diff
@@ -6,24 +6,44 @@ Feature: git prune-branches: remove stale feature branches when run on a feature
 
 
   Background:
-    Given I have a feature branch named "feature" behind main
-    And I have a feature branch named "stale_feature" behind main
-    And I am on the "feature" branch
+    Given the following commits exist in my repository
+      | BRANCH | LOCATION         | MESSAGE     | FILE NAME |
+      | main   | local and remote | main commit | main_file |
+    And I have a feature branch cut from "Initial commit" named "stale_feature_initial"
+    And I have a feature branch cut from "main commit" named "stale_feature_main"
```

Comment:
Same here. I dislike these branch names. I think just stale_feature1 and stale_feature2 will be fine.
Also have each be cut from initial commit (so each is behind the main branch).

May be good to add a test case for a branch that is even with the main branch (that may be a branch we just don't want to delete)